docs/spmd.md

```diff
@@ -0,0 +1,126 @@
+
```

Comment:
can you add this doc to docs/source/index.rst. so it will show up in https://pytorch.org/xla/release/2.0/index.html for 2.1 release